src/components/CondoCardModal.vue

```diff
@@ -74,14 +74,33 @@ export default {
       this.$store.commit('setActiveFeature', null);
     },
     addCondoRecords(state) {
-      if(this.$store.state.lastSearchMethod === "geocode") {
+      let mapUnitIds = function(id) {
         var i = 0;
-        this.$store.state.condoUnits.units[this.$store.state.activeCondo.featureId].map(
-          a => {a._featureId = a.properties.pwd_parcel_id + "-UNIT-" + i, i++}
+        let unitsToAdd = this.$store.state.condoUnits.units[id]
+        unitsToAdd.map(
+          a => {
+            typeof a.properties != 'undefined' ? a._featureId = a.properties.pwd_parcel_id + "-UNIT-" + i :
+            a._featureId = a.pwd_parcel_id + "-UNIT-" + i, i++
+          }
         );
-        this.$store.commit('setGeocodeRelated', this.$store.state.condoUnits.units[this.$store.state.activeCondo.featureId]);
-      } else { console.log("lastSearchMethod: ", this.$store.state.lastSearchMethod);}
-      this.$controller.dataManager.fetchData();
+        return unitsToAdd
+      }
+      mapUnitIds = mapUnitIds.bind(this)
```

Comment:
I don't think that you actually need this at all if you do it inline or as part of vue methods like I suggested earlier, but you can also define a function using arrow notation () => {} to autobind this to the method.

Reply:
I think its an either or. If its in the method, it might need a "this" when called or it needs to be in the function with "this" bound. 

Any advantage to the arrow notation? I've just sort of wrapped my head around "this" over the past year and seeing the bind function helped to understand.